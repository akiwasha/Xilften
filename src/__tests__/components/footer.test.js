import React from 'react';
import { render } from '@testing-library/react';
import { Footer } from '../../components';
import { Item } from '../../components/accordion/styles/accordion';

describe('<Footer />', () => {
  it('renders the <Footer /> with the populated data', () => {
    const { container, getByText } = render(
      <Footer>
        <Footer.Title>Des questions ? Appelez le 0805-220-512</Footer.Title>
        <Footer.Break />
        <Footer.Row>
          <Footer.Column>
            <Footer.Link href="#">FAQ</Footer.Link>
            <Footer.Link href="#">Relations Investisseurs</Footer.Link>
            <Footer.Link href="#">Modes de lecture</Footer.Link>
            <Footer.Link href="#">Mentions légales</Footer.Link>
            <Footer.Link href="#">Programmes originaux xilften</Footer.Link>
          </Footer.Column>

          <Footer.Column>
            <Footer.Link href="#">Centre d'aide</Footer.Link>
            <Footer.Link href="#">Recrutement</Footer.Link>
            <Footer.Link href="#">Conditions d'utilisation</Footer.Link>
            <Footer.Link href="#">Nous contacter</Footer.Link>
          </Footer.Column>

          <Footer.Column>
            <Footer.Link href="#">Compte</Footer.Link>
            <Footer.Link href="#">Utiliser des cartes cadeaux</Footer.Link>
            <Footer.Link href="#">Confidentialité</Footer.Link>
            <Footer.Link href="#">Test de vitesse</Footer.Link>
          </Footer.Column>

          <Footer.Column>
            <Footer.Link href="#">Presse</Footer.Link>
            <Footer.Link href="#">Acheter des cartes cadeaux</Footer.Link>
            <Footer.Link href="#">Préférences de cookies</Footer.Link>
            <Footer.Link href="#">Informations légales</Footer.Link>
          </Footer.Column>
        </Footer.Row>
        <Footer.Break />
        <Footer.Text>xilften France</Footer.Text>
      </Footer>
    );

    expect(getByText('Des questions ? Appelez le 0805-220-512'));
    expect(getByText('FAQ'));
    expect(getByText('Relations Investisseurs'));
    expect(getByText('Modes de lecture'));
    expect(getByText('Mentions légales'));
    expect(getByText('Programmes originaux xilften'));
    expect(container.firstChild).toMatchSnapshot();
  });
});
