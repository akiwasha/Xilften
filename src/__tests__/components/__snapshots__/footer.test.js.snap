// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders the <Footer /> with the populated data 1`] = `
<div
  class="footer__Container-qu6i6-0 hygnSq"
>
  <p
    class="footer__Title-qu6i6-4 ietrFk"
  >
    Des questions ? Appelez le 0805-220-512
  </p>
  <p
    class="footer__Break-qu6i6-6 cnYatT"
  />
  <div
    class="footer__Row-qu6i6-2 kLvuYU"
  >
    <div
      class="footer__Column-qu6i6-1 jAYoWp"
    >
      <a
        class="footer__Link-qu6i6-3 iRyBkK"
        href="#"
      >
        FAQ
      </a>
      <a
        class="footer__Link-qu6i6-3 iRyBkK"
        href="#"
      >
        Relations Investisseurs
      </a>
      <a
        class="footer__Link-qu6i6-3 iRyBkK"
        href="#"
      >
        Modes de lecture
      </a>
      <a
        class="footer__Link-qu6i6-3 iRyBkK"
        href="#"
      >
        Mentions légales
      </a>
      <a
        class="footer__Link-qu6i6-3 iRyBkK"
        href="#"
      >
        Programmes originaux Netflix
      </a>
    </div>
    <div
      class="footer__Column-qu6i6-1 jAYoWp"
    >
      <a
        class="footer__Link-qu6i6-3 iRyBkK"
        href="#"
      >
        Centre d'aide
      </a>
      <a
        class="footer__Link-qu6i6-3 iRyBkK"
        href="#"
      >
        Recrutement
      </a>
      <a
        class="footer__Link-qu6i6-3 iRyBkK"
        href="#"
      >
        Conditions d'utilisation
      </a>
      <a
        class="footer__Link-qu6i6-3 iRyBkK"
        href="#"
      >
        Nous contacter
      </a>
    </div>
    <div
      class="footer__Column-qu6i6-1 jAYoWp"
    >
      <a
        class="footer__Link-qu6i6-3 iRyBkK"
        href="#"
      >
        Compte
      </a>
      <a
        class="footer__Link-qu6i6-3 iRyBkK"
        href="#"
      >
        Utiliser des cartes cadeaux
      </a>
      <a
        class="footer__Link-qu6i6-3 iRyBkK"
        href="#"
      >
        Confidentialité
      </a>
      <a
        class="footer__Link-qu6i6-3 iRyBkK"
        href="#"
      >
        Test de vitesse
      </a>
    </div>
    <div
      class="footer__Column-qu6i6-1 jAYoWp"
    >
      <a
        class="footer__Link-qu6i6-3 iRyBkK"
        href="#"
      >
        Presse
      </a>
      <a
        class="footer__Link-qu6i6-3 iRyBkK"
        href="#"
      >
        Acheter des cartes cadeaux
      </a>
      <a
        class="footer__Link-qu6i6-3 iRyBkK"
        href="#"
      >
        Préférences de cookies
      </a>
      <a
        class="footer__Link-qu6i6-3 iRyBkK"
        href="#"
      >
        Informations légales
      </a>
    </div>
  </div>
  <p
    class="footer__Break-qu6i6-6 cnYatT"
  />
  <p
    class="footer__Text-qu6i6-5 bAqYDB"
  >
    Netflix France
  </p>
</div>
`;
